python/legion_linux/legion_linux/legion_gui.py

```diff
@@ -1424,9 +1427,8 @@ def __init__(self, controller:LegionController, icon:QtGui.QIcon):
         self.controller = controller
         self.controller.main_window = self
 
-        # Set a Minium Size to the window
-        # Scroll Area make window small square by default
-        self.setMinimumSize(1250, 1000)
```

Comment:
Revert this change your make it `self.setMinimumSize(1250, 725)`
![image](https://github.com/johnfanv2/LenovoLegionLinux/assets/68701049/6a2c0362-a094-4fa7-80e6-0a0a341c259f)